### ISSUE TITLE
Use up and down arrows for history in console adapter

### ISSUE THREAD

**Issue by keathley** [hedwig-im/hedwig#75]
This is something that I find myself doing out of habit. It would be really nice to use the up and down arrows to iterate through the previous command history to make it easier when developing responders.

**Comment by scrogson** [comment]
Yeah, this would be nice. We'll probably need to use the `tty_sl` driver for this?
